Packs/MaintenanceUpdateNotifications/IncidentTypes/incidenttype-Maintenance_Activity_Monitoring.json

```diff
@@ -0,0 +1,28 @@
+{
+    "autorun": true,
+    "color": "#3C64C5",
+    "days": 0,
+    "daysR": 0,
+    "default": false,
+    "detached": false,
+    "disabled": false,
+    "extractSettings": {
+        "fieldCliNameToExtractSettings": {},
+        "mode": "Specific"
+    },
+    "hours": 0,
+    "hoursR": 0,
+    "id": "Maintenance Activity Monitoring",
+    "layout": "Maintenance Activity Monitoring",
+    "locked": false,
+    "name": "Maintenance Activity Monitoring",
```

Comment:
the names might confuse the setup - maybe we should add `job set up` or something like that to make a good distinction between them? Also, remember that for now, your are supporting only XSOAR 8 - however you might want to add more notifications for other products (per user needs)...  so the name can be changed again :)